USER_MANUAL.md

```diff
@@ -55,5 +55,25 @@ Actors used in gyms are in `Contetnt\Actors`: add any new Actors to this directo
   2. Additionally a handover value is incremented to monitor how many times this is called.
   3. The stack count, and the handover counter are then checked they are the same.
 
+#### QA Workflows Testing
+
+##### ReplicatedStartupActors gym
+* For QA workflows Test Replicated startup actors are correctly spawned on all clients 
+* Validation
+  1. New Play Session is started 
+  2. After two seconds checks that actor is visible to client and reports pass or fail
+
+##### DestroyingActorsAtRuntime gym
+* For QA workflows Test Demonstrates that when a Level Actor is destroyed by server, a late connecting client does not see this Actor
+* Used to support QA test case "C1945 - Stably named actors can be destroyed at runtime and late-connecting clients don't see them"
+* Validation:
+  1. New Play session is started, at 10 seconds Actor is deleted and success message is shown notifying the PIE clients
+  2. Connect a client to Play session via command line
+  ```
+  sh
+  "C:\UnrealEngine\Engine\Binaries\Win64\UE4Editor.exe" "C:\UnrealGDKTestGyms\Game\GDKTestGyms.uproject" 127.0.0.1 -game -log -workerType UnrealClient -stdout -nowrite -unattended -nologtimes -nopause -noin -messaging -NoVerifyGC -windowed -ResX=800 -ResY=600
```

Comment:
Absolute path in a shell script in a doc.

Reply:
I can remove this as it's listed in the test case, apologies but wasn't sure what level of information to include :)